.github/workflows/dockerimage.yml

```
@@ -0,0 +1,31 @@
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+    env:
+      IMAGE_FULL: quay.io/eclipse/che-dashboard-next:next
```

It's not eclipse yet. 
```suggestion
      IMAGE_FULL: quay.io/che-incubator/che-dashboard-next:next
```

---

Good catch. Fixed.

---

Thanks